scripts/dev-e2e.sh

```diff
@@ -5,37 +5,28 @@ set -euo pipefail
 scripts="$(cd "$(dirname "$0")"; pwd)"
 source "$scripts"/init.sh
 
-AOHOME="$scripts/../../armada-operator"
 STATUSFILE="$(mktemp)"
 AOREPO='https://github.com/armadaproject/armada-operator.git'
 ARMADACTL_VERSION='0.19.1'
-JOB_DETAILS=1
+JOB_DETAILS="${JOB_DETAILS:-0}"
```

Comment:
```suggestion
JOB_DETAILS="${JOB_DETAILS:-1}"
```
Why wouldn't that be useful for a first-time invocation / default behaviour?